experimental/PropertyDDS/packages/property-common/src/datastructures/integer64.ts

```diff
@@ -100,6 +100,7 @@ function _stringToInt64(in_signed: boolean, in_string: string, in_radix = 10): n
     while (position < string.length) {
         const digit = parseInt(string[position++], in_radix);
         if (isNaN(digit)) {
+            // eslint-disable-next-line unicorn/prefer-type-error
```

Comment:
What's special about this one?

Reply:
Same question for the other couple of exceptions.

Reply:
I didn't look closely enough. I missed that this was checking the result of string parsing. This isn't special; I'll update.